srttools/monitor/webserver.py

```diff
@@ -298,43 +135,37 @@ def log_function(_):
 
         application.log_request = log_function
         self.web_server = tornado.httpserver.HTTPServer(application)
+        asyncio.set_event_loop(asyncio.new_event_loop())
         try:
-            self.web_server.listen(self.port)
+            self.web_server.listen(self.port, address=self.address)
         except OSError:
             raise OSError(f"Port {self.port} is already being used, choose a different one!")
 
     def start(self):
-        self._asyncioloop = None
-        try:
-            asyncio.get_event_loop()
-        except RuntimeError:
-            self._asyncioloop = asyncio.new_event_loop()
-            asyncio.set_event_loop(self._asyncioloop)
-        self.ioloop = tornado.ioloop.IOLoop.current()
+        create_index_file(port=self.port)
 
-        create_index_file(self.port)
-
-        self.t = threading.Thread(target=self.ioloop.start)
-        self.t.start()
+        self.ioloop = tornado.ioloop.IOLoop.current()
+        if not self.started and not self.ioloop.asyncio_loop.is_running():
+            self.t = threading.Thread(target=self.ioloop.start)
+            self.t.start()
```

Comment:
Fix for asyncio that was complaining about some loop that was already started, something like that.